jaxb/pom.xml

```diff
@@ -54,6 +54,14 @@ for configuring data-binding.
       <artifactId>jackson-databind</artifactId>
     </dependency>
 
+    <!-- Tolerate [2.2,3) for javax.xml.bind Import-Package in the generated MANIFEST.MF -->
+    <!-- https://mvnrepository.com/artifact/javax.xml.bind/jaxb-api -->
+    <dependency>
```

Comment:
Hmmh. Would this not cause problems wrt dependency that follows, for `jakarta.xml.bind-api:2.3.3`, including essentially duplicate version?